host/sgx/sgxquoteprovider.c

```diff
@@ -337,3 +340,239 @@ void oe_free_qe_identity_info_args(oe_get_qe_identity_info_args_t* args)
             free(args->host_out_buffer);
     }
 }
+
+oe_result_t oe_get_quote_verification_collateral(
```

Comment:
add sgx into function name for consistency with line 345

Reply:
Thanks, function name changed.